google/cloud/bigtable/doc/bigtable-samples-instance-admin.dox

```diff
@@ -0,0 +1,65 @@
+/*!
+@page bigtable-samples-instance-admin Example: C++ Instance Admin Samples
+
+This example is a very simple "Instance Admin" application, written in C++, that
+illustrates how to:
+
+- Creates an instance(type: PRODUCTION) and run basic instance-operations.
+- Create a development instance.
+- Delete an instance
+- Create a new cluster.
+- Delete a cluster.
+
+## Running the example
+
+This example uses the
+[Cloud Bigtable C++ Client Library](https://GoogleCloudPlatform.github.io/google-cloud-cpp)
+to communicate with Cloud Bigtable.
+
+To run the example program, follow the
+[instructions](https://github.com/GoogleCloudPlatform/google-cloud-cpp/tree/master/google/cloud/bigtable/examples/)
+for the example on GitHub.
+
+### Including the Necessary Headers
+
+The example uses the following headers:
+
+@snippet bigtable_samples_instance_admin.cc bigtable includes
+
+### Connect to the Cloud Bigtable admin endpoint
+
+@snippet bigtable_samples_instance_admin.cc connect instance admin client 
+
+### Connect to the Cloud Bigtable endpoint.
+
+@snippet bigtable_samples_instance_admin.cc connect instance admin 
+
+### Creates an instance(type: PRODUCTION) and run basic instance-operations.  
```

Comment:
Maybe "Create a production instance and ..."

Reply:
fixed.